src/client/src/core/util/validation/rules.ts

```diff
@@ -70,14 +70,17 @@ export const shouldNotExceedCharLength = function(charLength: number) {
   };
 };
 
-export const validDateFormat = function(dateFormat = 'yyyy-MM-dd') {
+export const validDateFormat = function(
+  displayFormat = 'yyyy-mm-dd',
```

Comment:
@Super-Chama just an idea. can we move this rules to composable. so we can use `userDateFormat` from `@/core/util/composable/useDateFormat` inside rules and avoid passing them from vue classes as `displayFormat` is a global constant. 
If we go for composable, we have to change all vue files which contains validation right?
This is not for this release. but if you feel it's a good idea will go for it later.

Reply:
Yeah that's a good idea. all validations as composable right? we'll have to do heavy refactoring tho :face_with_head_bandage: